Dockerfile

```diff
@@ -73,6 +73,8 @@ RUN go install github.com/infobloxopen/protoc-gen-atlas-query-validate
 # Download all dependencies of protoc-gen-atlas-validate
 RUN cd ${GOPATH}/src/github.com/infobloxopen/protoc-gen-atlas-validate && dep ensure -vendor-only
 RUN go install github.com/infobloxopen/protoc-gen-atlas-validate
+RUN go get github.com/envoyproxy/protoc-gen-validate
+RUN go install github.com/envoyproxy/protoc-gen-validate
```

Comment:
1) do we need go get here? propose to remove it
2) lets put go install at line 63 (after line 62)

Reply:
we don't need go install, because go get will call install itself :
```
Get downloads the packages named by the import paths, along with their
dependencies. It then installs the named packages, like 'go install'.
```